app/views/directives/process-template-dialog/process-template-info.html

```diff
@@ -0,0 +1,20 @@
+<div class="order-service-details" ng-if="!$ctrl.selectStep.selected">
```

Comment:
I don't think the `ng-if` is needed here because you check `ng-if="step.selected"` above before including the view.